test/e2e/set/remotekubernetescluster/remotekubernetescluster_finalizer.go

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) 2024 ScyllaDB.
+
+package remotekubernetescluster
+
+import (
+	"context"
+	"fmt"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	scyllav1alpha1 "github.com/scylladb/scylla-operator/pkg/api/scylla/v1alpha1"
+	"github.com/scylladb/scylla-operator/pkg/controllerhelpers"
+	"github.com/scylladb/scylla-operator/pkg/helpers/slices"
+	"github.com/scylladb/scylla-operator/test/e2e/framework"
+	"github.com/scylladb/scylla-operator/test/e2e/utils"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = g.Describe("RemoteKubernetesCluster finalizer", func() {
+	f := framework.NewFramework("remotekubernetescluster")
+
+	g.It("should block deletion when there are referencing ScyllaDBClusters", func() {
```

Comment:
```suggestion
	g.It("should block deletion when there are ScyllaDBClusters referencing the object", func() {
```
or ScyllaDBCluster referents?
nit

Reply:
done